### ISSUE TITLE
Sustained Scenario Run Harness

### ISSUE THREAD

**Issue by jsquire** [Azure/azure-sdk-for-net#8194]
# Summary

To ensure that the track two client library performs consistently over an extended period of time and does not exhibit instability or excess resource consumption, a stress harness is needed to allow running core scenarios repeatedly for an extended period of time.  The focus of this testing to validate data integrity and stability over time rather than raw performance.

# Scope of Work

- Create a stand-alone application which can be hosted on a virtual machine or in a container which will exercise the core usage scenarios for the Event Hubs client libraries repeatedly for an extended period.

- Scenarios run looped, ideally with some randomization to the data that they are using to generate events.

- Scenarios should be self-contained and execute in parallel, to a reasonable degree.

# Out of Scope

- Collection of performance-based metrics.  

- Advanced diagnostics, inspection, or reporting; this is a minimal harness and not intended to be an industrial-strength product.

# Success Criteria

- The harness has been created and added to a scratch area of GitHub, separate from the main repository.

- The harness can be executed and has proven stable over a period of a working day such that we have confidence any problems are the result of our Event Hubs client and not the harness itself.
